### PR TITLE
[SOAR-4029] - Hunt_for_Threats_in_DNS_Logs_with_InsightIDR_from_Slack

### PR DESCRIPTION
**Pull request by jmcadams-r7**
## Description: 

This PR adds the following WF:
Find_Indicator_in_DNS_Logs_with_InsightIDR_from_Slack

## Validation: 
```
RMT-MBP-5357:Find_Indicator_In_DNS_Logs_with_InsightIDR_from_Slack jmcadams$ icon-validate .
[*] Validating workflow at .

[*] Running Integration Validators...
[*] Executing validator WorkflowDirectoryNameMatchValidator
[*] Executing validator WorkflowFilesValidator
[*] Executing validator WorkflowHelpValidator
[*] Executing validator WorkflowChangelogValidator
[*] Executing validator WorkflowVendorValidator
[*] Executing validator WorkflowVersionValidator
[*] Executing validator WorkflowExtensionValidator
[*] Executing validator WorkflowSupportValidator
[*] Executing validator WorkflowPNGHashValidator
[*] Executing validator WorkflowICONFileNameValidator
[*] Executing validator WorkflowScreenshotValidator
[*] Executing validator WorkflowTitleValidator
[*] Executing validator WorkflowDescriptionValidator
[*] Executing validator WorkflowNameValidator
[*] Executing validator WorkflowProfanityValidator
[*] Executing validator WorkflowHelpPluginUtilizationValidator
[*] Executing validator WorkflowICONFileValidator
[*] Workflow successfully validated!

----
[*] Total time elapsed: 24.284ms
```

slack_message_found
slack_message_not_found
workflow1
workflow2

### CODE REVIEW

File: workflows/Find_Indicator_in_DNS_Logs_with_InsightIDR_from_Slack/workflow.spec.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+extension: workflow
+products: ["insightconnect"]
+name: Find_Indicator_in_DNS_Logs_with_InsightIDR_from_Slack
+title: "Find Indicator in DNS Logs with InsightIDR from Slack"
```
Comment (jschipp-r7) [inline workflows/Find_Indicator_in_DNS_Logs_with_InsightIDR_from_Slack/workflow.spec.yaml:4]: Recommend renaming workflow and all references

```suggestion
title: "Hunt for Threat Indicators in InsightIDR DNS logs from Slack"
```

File: workflows/Find_Indicator_in_DNS_Logs_with_InsightIDR_from_Slack/help.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
+# Description
+
+Lookup suspicious URLs in IDR DNS Query logs with a simple slack command.
```
Comment (jschipp-r7) [inline workflows/Find_Indicator_in_DNS_Logs_with_InsightIDR_from_Slack/help.md:3]:
```suggestion
Lookup suspicious URLs or domains in IDR DNS Query logs with a simple slack command.
```

File: workflows/Find_Indicator_in_DNS_Logs_with_InsightIDR_from_Slack/help.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+Sample Slack Trigger Commands:
+
+`@Rapid7 InsightConnect get-logs aadroid.net`
+
+# Key Features
+
+* **Quickly analyze the scope of a malicious URL** - Quickly find out how many of your users were affected by a malicious URL
```
Comment (jschipp-r7) [inline workflows/Find_Indicator_in_DNS_Logs_with_InsightIDR_from_Slack/help.md:11]:
```suggestion
* **Quickly analyze the scope of a malicious URL** - Quickly find out how many of your users were affected by a malicious URL or domain
```

File: workflows/Find_Indicator_in_DNS_Logs_with_InsightIDR_from_Slack/help.md (hunk 2 above)
Comment (jschipp-r7) [reply workflows/Find_Indicator_in_DNS_Logs_with_InsightIDR_from_Slack/help.md:3]: Also, document how we match either URL or domain so the user understands how it works

**Review by jschipp-r7 — CHANGES_REQUESTED** [review]

**Review by cstuart-r7 — APPROVED** [review]
LGTM

File: workflows/Hunt_for_Threats_in_DNS_Logs_with_InsightIDR_from_Slack/help.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
+# Description
+
+Lookup suspicious domains in IDR DNS Query logs with a simple slack command.
```
Comment (jschipp-r7) [inline workflows/Hunt_for_Threats_in_DNS_Logs_with_InsightIDR_from_Slack/help.md:3]:
```suggestion
Lookup suspicious domains in IDR DNS Query logs with a simple slack command. This workflow accepts domains and URLs as input.
```

File: workflows/Hunt_for_Threats_in_DNS_Logs_with_InsightIDR_from_Slack/help.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+Sample Slack Trigger Commands:
+
+`@Rapid7 InsightConnect get-logs aadroid.net`
+
```
Comment (jschipp-r7) [inline workflows/Hunt_for_Threats_in_DNS_Logs_with_InsightIDR_from_Slack/help.md:8]:
```suggestion

`@Rapid7 InsightConnect get-logs https://aadroid.net`

```

**Review by jschipp-r7 — APPROVED** [review]

**Pull request merged**